python-scripts/README.md

```diff
@@ -3,19 +3,7 @@
 This is a collection of scripts to access various Xbox hardware.
 
 
-## Usage for updated scripts
+## Use
```

Comment:
nit: Would prefer to have a slightly longer section title like "How to Use" or something

Reply:
Will do and then merge. Thanks!